projects/packages/my-jetpack/_inc/data/use-simple-mutation.ts

```diff
@@ -0,0 +1,39 @@
+import { useMutation } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import { useFetchingErrorNotice } from './notices/use-fetching-error-notice';
+import type { UseMutationOptions, UseMutationResult } from '@tanstack/react-query';
+import type { APIFetchOptions } from '@wordpress/api-fetch';
+
+interface Options extends UseMutationOptions {
```

Comment:
```suggestion
interface Options<T> extends UseMutationOptions {
```

I would avoid `unknown` types in any case if possible.